docs/changelog.txt

```diff
@@ -1,3 +1,13 @@
+0.5.3 (2020-07-05)
+------------------------------------------------------------------------
+This release primarily restores legacy industry properties and vars which were removed in NML 0.5.0.  This is done for compatibility, so that older industry grfs can still be compiled with NML 0.5.3.  
+
+See the NML 0.5.0 changelog entry for a list of the legacy industry properties and vars that were removed, and are restored in NML 0.5.3.
```

Comment:
... A warning will be shown when these are used, and they may be removed again in some future version.